shell/browser/ui/views/win_frame_view.cc

```diff
@@ -34,6 +34,13 @@ void WinFrameView::Init(NativeWindowViews* window, views::Widget* frame) {
   window_ = window;
   frame_ = frame;
 
+  // Prevent events from trickling down the views hierarchy here, since
+  // when a given window is frameless we only want to use FramelessView's
+  // ResizingBorderHitTest in ShouldDescendIntoChildForEventHandling.
+  // See https://chromium-review.googlesource.com/c/chromium/src/+/3251980.
+  if (!window_->has_frame())
```

Comment:
Should this only be called for `window_->IsResizable()` frameless windows ?

Reply:
oh good thought 🤔  probably? mind's well be as specific as possible.

Reply:
updated.